core/jazz_admin-api/config/dev-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ADMIN_ID": "{jazz_admin}",
+  "PRIVATE_TOKEN_GITLAB": "{private-token}",
+  "BB_USERNAME":"{bb_username}",
+  "BB_PASSWORD":"{bb_password}",
+  "BASE_URL":"{base-url}",
+  "SCM_TYPE": "{scm-type}",
+
+  "PATH_GIT": "/slf/jazz-build-module/raw/master/jazz-installer-vars.json",
+  "PATH_BITBUCKET":"/projects/SLF/repos/jazz-build-module/raw/jazz-installer-vars.json"
```

Comment:
Can we use GITLAB_PATH and BITBUCKET_PATH in line 9 and 10 respectively? And same changes in other config.json file